data/mods/Arcana/monsters.json

```diff
@@ -677,7 +677,7 @@
         ["TENTACLE", 5]
     ],
     "description":"A large plantlike creature with a gaping, toothy maw and numerous vines.  A hideous smell permeates the air around it.",
-    "flags":["HEARS", "SMELLS", "BASHES", "NOHEAD", "POISON", "NO_BREATHE", "PARALYZEVENOM"]
+    "flags":["HEARS", "SMELLS", "BASHES", "NOHEAD", "POISON", "NO_BREATHE", "PARALYZEVENOM", "PATH_AVOID_DANGER_2", "PRIORITIZE_TARGETS"]
     },{
```

Comment:
Used by chicken walkers, police bots, eyebots, grackens, cleaner bots, mi-gos, riot control bots, Shia LaBeouf, tank drones, tribots, searchlights, zombie masters, zombie necromancers, and zombie scientists.

I am unsure what the full effects are, in detail. But it seems primarily used by intelligent monsters and robots. Of these, the archon is the one most likely intended to be intelligent. I added this to the moroboru in the hopes it will further mitigate any potential for infighting, but it may not be needed.